test/test_faiss_and_milvus.py

```diff
@@ -192,6 +194,23 @@ def test_finding(document_store, retriever):
     assert len(prediction.get('documents', [])) == 1
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize("retriever", ["dpr"], indirect=True)
+@pytest.mark.parametrize("document_store", ["faiss", "milvus"], indirect=True)
+@pytest.mark.parametrize("batch_size", [4, 6])
+def test_delete_docs_with_filters(document_store, retriever, batch_size):
```

Comment:
Why do you parametrize the `batch_size` here? In this current setup, we will run the test 4x. Can we drop the batch_size so that it is only 2x?